frontend/src/metabase/visualizations/components/settings/ChartSettingTableColumns/TableColumnPanel/TableColumnPanel.unit.spec.tsx

```diff
@@ -103,6 +103,10 @@ describe("DatasetColumnSelector", () => {
     expect(items[1]).toHaveTextContent("ID");
     expect(items[2]).toHaveTextContent("Tax");
     expect(items[3]).toHaveTextContent("Subtotal");
+
+    items.forEach(item =>
+      expect(within(item).getByLabelText("More info")).toBeInTheDocument(),
+    );
```

Comment:
This does not match the test name (`should display columns in the order of the setting`) - let's add a separate test for it.